### PR TITLE
Asman Umbetov Task 81-90

### PR DESCRIPTION
**Pull request by fastndead**

### CODE REVIEW

File: tasks/85_bet-validation/bet-validation.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Any other character makes the bet invalid.
+ * @example
+ * // returns [1, 2, 3, 4, 5]
+ * validate_bet('1 2 3 4 5)
+ * @param {Number} N - number of numbers needed
+ * @param {Number} M - max number allowed
+ * @param {String} message
+ * @return {Array}
+ */
+// eslint-disable-next-line camelcase
```
Comment (vovcyan) [inline tasks/85_bet-validation/bet-validation.js:15]: Do not use it

File: tasks/86_working-hours/working-hours.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        "day": "sun",
+        "from": "11:00",
+        "to": "23:00"
+    }
+ ];)
+ * @param {Array<Object>} data
+ * @return {Array<String>}
+ */
+export function formatWorkingHours(data) {
+  data.sort((day1, day2) => {
```
Comment (vovcyan) [inline tasks/86_working-hours/working-hours.js]: Pure function

File: tasks/88_record-collection/record-collection.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+ *    'album': 'ABBA Gold',
+ *  },
+ * };)
+ * @param {String} id - id of the object in collection
+ * @param {String} prop - property of object
+ * @param {String} value
+ * @return {Array<Object>} modified collection
+ */
+export function updateRecords(id, prop, value ) {
+  let collectionCopy = JSON.parse(JSON.stringify(collection));
```
Comment (vovcyan) [inline tasks/88_record-collection/record-collection.js]: const

File: tasks/89_symmertic-difference/symmertic-difference.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ * diffArray(["diorite", "andesite", "grass", "dirt", "pink wool",
+ * "dead shrub"], ["diorite", "andesite", "grass", "dirt", "dead shrub"])
+ * @param {Array} arr1 - first array
+ * @param {Array} arr2 - second array
+ * @return {Array} - symmetric difference of the two arrays
+ */
+export function diffArray(arr1, arr2) {
+  const arr1Copy = Array.from(new Set(arr1));
+  const arr2Copy = Array.from(new Set(arr2));
+  const result = [...arr1Copy.filter((item) => !arr2Copy.includes(item)),
```
Comment (vovcyan) [inline tasks/89_symmertic-difference/symmertic-difference.js]: Split it

File: tasks/90_inventory-update/inventory-update.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * @example
+ * // returns [[42, "Bowling Ball"], [2, "Dirty Sock"]
+ * updateInventory([[21, "Bowling Ball"], [2, "Dirty Sock"]],
+ * [[21, "Bowling Ball"]],)
+ * @param {Array} currentInv
+ * @param {Array} newInv
+ * @return {Array}
+ */
+export function updateInventory(currentInv, newInv) {
+  const resultInv = currentInv;
```
Comment (vovcyan) [inline tasks/90_inventory-update/inventory-update.js]: For what do you needs this?
If you wanna reassign array,  please use Array.from

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: tasks/85_bet-validation/bet-validation.js (hunk 1 above)
Comment (fastndead) [reply tasks/85_bet-validation/bet-validation.js:15]: didn't we discuss it? eslint marks the name of function as an error, what should I do then?

File: tasks/90_inventory-update/inventory-update.js (hunk 5 above)
Comment (fastndead) [reply tasks/90_inventory-update/inventory-update.js]: i used the new array to prevent mutation of the one, that comes as parameter.

**Review by akimovaa — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**